docs/source/geopmwrite.1.rst

```diff
@@ -190,9 +203,15 @@ Set all CPUs on package 0 to 1.5 *GHz* (cpu 1 is on package 0):
    $ geopmread CPU_FREQUENCY_MAX_CONTROL cpu 1
    1.5e9
 
+.. code-block::
```

Comment:
```suggestion
Set the uncore frequency of all CPUs to 1.5 GHz. Perform the operation in a single GEOPM batch-write request instead of across multiple `geopmwrite` commands:

.. code-block::
```